### ISSUE TITLE
[BUG] [Linter] rule H021 (inline styles) broken for multiline tags

### ISSUE THREAD

**Issue by kleins05** [Riverside-Healthcare/djLint#678]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: mac OS 10.13
 - Python Version: 3.11.2
 - djLint Version: 1.30.0
 - template language: django

## Issue
If a tag with a style attribute spans multiple lines, rule H021 does not detect the inline style.

Example:
```
<div style="color:green"
     class="foo">
</div>
```
Same example with the whole div tag in one line works.

## How To Reproduce
1. Create a template file `test.html` with the above example. `djlint test.html` lints successfully. Should fail.
2. Place the whole opening tag into one line (move the class attribute). `djlint test.html` fails with rule H021. Correct

## Contents of .djlintrc/pyproject.toml [tool.djlint]
ignore = "H017,H031"
indent = 2
quiet = true
